scripts/exregional_process_radarref.sh

```diff
@@ -0,0 +1,348 @@
+#
+# Get the full path to the file in which this script/function is located 
+# (scrfunc_fp), the name of that file (scrfunc_fn), and the directory in
+# which the file is located (scrfunc_dir).
+#
+#-----------------------------------------------------------------------
+#
+scrfunc_fp=$( readlink -f "${BASH_SOURCE[0]}" )
+scrfunc_fn=$( basename "${scrfunc_fp}" )
+scrfunc_dir=$( dirname "${scrfunc_fp}" )
+#
+#-----------------------------------------------------------------------
+#
+# Print message indicating entry into script.
+#
+#-----------------------------------------------------------------------
+#
+print_info_msg "
+========================================================================
+Entering script:  \"${scrfunc_fn}\"
+In directory:     \"${scrfunc_dir}\"
+This is the ex-script for the task that runs radar reflectivity preprocess
+with FV3 for the specified cycle.
+========================================================================"
+#
+#-----------------------------------------------------------------------
+#
+# Load modules.
+#
+#-----------------------------------------------------------------------
+#
+eval ${PRE_TASK_CMDS}
+
+nprocs=$(( NNODES_PROCESS_RADARREF*PPN_PROCESS_RADARREF))
+
+#gridspec_dir=${NWGES_BASEDIR}/grid_spec
+#
+#-----------------------------------------------------------------------
+#
+# Extract from CDATE the starting year, month, day, and hour of the
+# forecast.  These are needed below for various operations.
+#
+#-----------------------------------------------------------------------
+#
+#START_DATE=$(echo "${CDATE}" | sed 's/\([[:digit:]]\{2\}\)$/ \1/')
+START_DATE=$(echo "${PDY} ${cyc}")
+YYYYMMDDHH=$(date +%Y%m%d%H -d "${START_DATE}")
+#JJJ=$(date +%j -d "${START_DATE}")
+
+YYYY=${YYYYMMDDHH:0:4}
+MM=${YYYYMMDDHH:4:2}
+DD=${YYYYMMDDHH:6:2}
+HH=${YYYYMMDDHH:8:2}
+#YYYYMMDD=${YYYYMMDDHH:0:8}
+
+#
+#-----------------------------------------------------------------------
+#
+# Find cycle type: cold or warm 
+#  BKTYPE=0: warm start
+#  BKTYPE=1: cold start
+#
+#-----------------------------------------------------------------------
+#
+BKTYPE=0
+if [ ${DO_SPINUP} == "TRUE" ]; then
+  if [ ${CYCLE_TYPE} == "spinup" ]; then
+    for cyc_start in "${CYCL_HRS_SPINSTART[@]}"; do
+      if [ ${HH} -eq ${cyc_start} ]; then
+        BKTYPE=1
+      fi
+    done
+  fi
+else
+  for cyc_start in "${CYCL_HRS_PRODSTART[@]}"; do
+    if [ ${HH} -eq ${cyc_start} ]; then
+        BKTYPE=1
+    fi
+  done
+fi
+
+n_iolayouty=$(($IO_LAYOUT_Y-1))
+list_iolayout=$(seq 0 $n_iolayouty)
+
+#
+#-----------------------------------------------------------------------
+#
+# Loop through different time levels
+# Get into working directory
+#
+#-----------------------------------------------------------------------
+#
+print_info_msg "$VERBOSE" "
+Getting into working directory for radar reflectivity process ..."
+
+for bigmin in ${RADARREFL_TIMELEVEL[@]}; do
```

Comment:
bigmin is very difficult to understand from a readability perspective. Maybe the variable name `timelevel` would be clearer given the description of that variable? At least them we can recall that we're looping through the RADARREFL_TIMELEVEL list.